src/azure-cli-core/azure/cli/core/cloud.py

```diff
@@ -190,7 +190,7 @@ def __str__(self):
         sql_management='https://management.core.usgovcloudapi.net:8443/',
         batch_resource_id='https://batch.core.usgovcloudapi.net/',
         gallery='https://gallery.usgovcloudapi.net/',
-        active_directory='https://login.microsoftonline.com',
+        active_directory='https://login.microsoftonline.us',
```

Comment:
Was this something a git rebase reversed or had this just not yet been merged into the branch?

Reply:
It's in dev branch at https://github.com/Azure/azure-cli/commit/06cd9c2160c2f792a3af76f71fb55368c92cf0d6